client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala

```diff
@@ -205,7 +208,16 @@ class DeltaSharingRestClient(
     val encodedTableName = URLEncoder.encode(table.name, "UTF-8")
     val target = getTargetUrl(
       s"/shares/$encodedShareName/schemas/$encodedSchemaName/tables/$encodedTableName/metadata")
-    val (version, lines) = getNDJson(target)
+    val (version, respondedFormat, lines) = getNDJson(target)
+    if (responseFormat != respondedFormat) {
+      logWarning(s"RespondedFormat($respondedFormat) is different from requested responseFormat(" +
```

Comment:
Perhaps add a comment that this can happen if the DS server does not support the requested format.

Ideally, this should only happen if we request delta format but get back parquet. The reverse is not expected right?

Reply:
Right!

And this happens on the two recipient on uc use cases, where the delta sharing server is an open servers, and recipients are on databricks DBR, so recipients would any way use this code to ask for "delta" format, but may get parquet in response. And we should still be able to handle that case.